modules/ducktests/tests/ignitetest/services/utils/ignite_persistence.py

```diff
@@ -43,8 +44,7 @@ def init_persistent(self, node):
         Init persistent directory.
         :param node: Service node.
         """
-        node.account.mkdirs(self.PERSISTENT_ROOT)
-        node.account.mkdirs(self.TEMP_DIR)
+        node.account.mkdirs(f'{self.TEMP_DIR} {self.LOGS_DIR}')
```

Comment:
Why this change?

Reply:
create folders with one command, it's faster